### PR TITLE
464 refactoring input components new3

### LINKED ISSUE #464: Refactoring Input Components
**Refactoring Input Components**

We should substitute all Input components. 

We should particularly for Error-Input state work on specific changes. 

Design Material: https://www.figma.com/file/ZTqF54s6CSHTwAxp0U0JjQ/Circles-myxogastria?node-id=6783%3A10747

### PR DESCRIPTION
**Pull request by mikozet**

### CODE REVIEW

**Comment by louilinn** [comment]
I do not see this state, it seems only hoover and active state is used in the app for error message version?

File: src/components/Input.js [hunk 1]
```diff
@@ -4,34 +4,163 @@ import {
...
-
 const useStyles = makeStyles((theme) => ({
   inputLabel: {
+    transform: 'translate(28px, -6px)',
+    background: theme.custom.colors.white,
+    fontSize: '12px',
+    padding: '2px 6px',
+    transition: 'none',
+    color: (isOrganization) =>
+      isOrganization
```
Comment (louilinn) [inline src/components/Input.js]: these things are repeated a lot. Please define constants for color based on organization and DRY :)

File: src/components/Input.js [hunk 2]
```diff
@@ -4,34 +4,163 @@ import {
...
     },
-  },
-  inputUnderline: {
-    '&.Mui-focused::after': {
-      borderBottomColor: theme.palette.text.primary,
+
+    '&:has(+ .MuiOutlinedInput-root:hover)': {
+      background: (isOrganization) =>
+        isOrganization
+          ? theme.custom.colors.cornflowerBlue
```
Comment (louilinn) [inline src/components/Input.js]: Also repeated code

**Review by louilinn — CHANGES_REQUESTED** [review]
Great job! Works as intended :) except one comment I already made in this PR. Some small DRY code comments only

**Comment by mikozet** [comment]
>I do not see this state, it seems only hoover and active state is used in the app for error message version?
Uploading Screenshot from 2022-10-14 15-09-15.png…

The link is broken for me but I think I know what do you mean and I apply change - default state for error Input

**Comment by mikozet** [comment]
Hi @louilinn It seems like I didn't push changes before but now they are there ready to review :)

Sorry I still don't see those changes - I am checking now what have happened

**Comment by mikozet** [comment]
> Hi @louilinn It seems like I didn't push changes before but now they are there ready to review :)
> 
> Sorry I still don't see those changes - I am checking now what have happened

Forgot to write - all is good now :)

**Comment by louilinn** [comment]
@mikozet I talked to Rucha and found out that in the latest version on figma, also the search field has this design, so all input fields should have this design so 
- [x] replace search input field
- [x] remove any legacy code connected to the old input field

**Comment by louilinn** [comment]
Also I think something happened.. The hover state in the design is now the active state and the active state is never used.
Current behaviour
writing in field ("active") - has hover design (incorrect)
hovering on field - has hover design (correct)
leaving the field to click on other things and not hovering - has default state (correct)

This is the case for purple (shared wallet) and pink (error) but not blue (private wallet)

**Comment by louilinn** [comment]
I think we can leave the search field update for now but please fix this asap so we can demo:

> Also I think something happened.. The hover state in the design is now the active state and the active state is never used. Current behaviour writing in field ("active") - has hover design (incorrect) hovering on field - has hover design (correct) leaving the field to click on other things and not hovering - has default state (correct)
> 
> This is the case for purple (shared wallet) and pink (error) but not blue (private wallet)

**Comment by mikozet** [comment]
Hi @louilinn it is ready again to review. Bear in mind that input can be active + hovered or just active.

**Review by louilinn — COMMENTED** [review]
So I'm not sure but feels like if you enter the finder view from shared wallet it should have shared wallet theme?
Or do we always use blue for search?

**Comment by mikozet** [comment]
> So I'm not sure but feels like if you enter the finder view from shared wallet it should have shared wallet theme? Or do we always use blue for search?

You are right it is corrected now

**Pull request merged**